chemprop/models/ffn.py

```diff
@@ -0,0 +1,389 @@
+from typing import List, Tuple
+
+import torch
+import torch.nn as nn
+import re
+
+
+class AttrProxy(object):
```

Comment:
```suggestion
class AttrProxy:
```

Reply:
is this class used anywhere else? If not, then it would be better served as just a helper method inside the `MultiReadout` class.

Reply:
It really looks like you're just hacking the `__getattr__` method of your `ffn` attribute below so its indexable like a `list`. You can just wrap the `ffn` inside a `ModuleList` object if you want that behavior

Reply:
Thanks for this valuable suggestion. Indeed, using `ModuleList` is more clear. I have made the change.